### PR TITLE
Fixed bug in scenario from csv ruptures

### PR DESCRIPTION
**Pull request by micheles**
`number_of_ground_motion_fields` was ignored. Also added a check on sum_losses vs tot_losses in post_risk and added a test.

### CODE REVIEW

File: openquake/qa_tests_data/scenario/case_13/job.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [general]
 
 description = Multi-rupture scenario
-calculation_mode = scenario
+calculation_mode = event_based
```
Comment (raoanirudh) [inline openquake/qa_tests_data/scenario/case_13/job.ini:4]: Why the change in the `calculation_mode` here?

Comment (micheles) [reply openquake/qa_tests_data/scenario/case_13/job.ini:4]: Because it does not work for scenario yet, there will be another PR for that (the case of ruptures of different TRTs).
